app/views/taxons/_organisation_logos_and_list.html.erb

```diff
@@ -12,7 +12,7 @@
 </div>
 
 <% if organisations_without_logos.any? %>
-  <div>
+  <div <%= "data-module=track-click" if track_click?(organisations_without_logos) %>>
```

Comment:
I wonder if this could be simplified - instead of checking each bit of the page and adding the data module attribute, could it be applied just once to a higher level element? On the education page example there's 71 instances of it.

Reply:
I don't think that's linked to this line of code specifically? I think it's because we wanted the organisation logo component in static to be self-contained, e.g: contain the `data-module=track-click` itself so that it can be easily used by other people who need to add tracking without having to add that data-module themselves (like the problem we have with the document list).

However, this does mean that every time we use the logo, it comes with its own track-click... Hence the 71 track clicks.

I wasn't really happy with that solution but couldn't think of any alternative at the time. I guess we could have an argument passed to the component saying whether we're providing our own data-module wrapper or not?

Reply:
Added a card for this: https://trello.com/c/VSi9cy7j/132-improve-track-click-functionality-for-organisation-logo-component